jobs/JGLOBAL_FORECAST

```diff
@@ -59,6 +59,11 @@ export pgmerr=errfile
 ##############################################
 export CDATE=${CDATE:-${PDY}${cyc}}
 export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+
```

Comment:
Should this have if statements as in the develop branch: https://github.com/NOAA-EMC/global-workflow/blob/develop/jobs/JGLOBAL_FORECAST#L73-L79

Reply:
I think it should be OK either way. These variables already appear in JJOB so I'm not sure why they are here at all. But I included them just in case there's some other concerns that I'm not yet aware of. 

These scripts probably need a extensive clean up of excess variables, seems like.